tests/track_duration.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/bash
+
+# Stop script on NZEC
+set -e
+
+# Exposing stream 3 as a pipe to standard output of the script itself
+exec 3>&1
+
+#
+error_output() {
+    printf "%b\n" "${red:-}$script_name: Error: $1${normal:-}" >&2
+}
+
+output() {
+    printf "%b\n" "${cyan:-}$script_name:${normal:-} $1" >&3
+}
+
+verbose_output() {
+    if [ "$verbose" = true ]; then
+        output "$1"
+    fi
+}
+
+# This platform list is finite - if the SDK/Runtime has supported Linux distribution-specific assets,
+#
```

Comment:
i don't see a platform list, is this comment still relevant?

Reply:
good catch - it's no longer relevant since we chose the platform neutral route. TY.